src/test/java/uk/gov/justice/digital/delius/service/CourtAppearanceServiceTest.java

```diff
@@ -52,7 +44,11 @@ public void setUp() {
                         .softDeleted(1L)
                         .appearanceDate(LocalDateTime.now())
                         .offenderId(1L)
-                        .eventId(50L)
```

Comment:
All these tests change to reflect the Model changes and removal of service calls...